package/gluon-check-connection/Makefile

```diff
@@ -0,0 +1,13 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=gluon-check-connection
+PKG_VERSION:=1
+
+include ../gluon.mk
+
+define Package/gluon-check-connection
+  TITLE:=Checks if a node can ping definable targets
```

Comment:
like in other contributions, i don't think the "Title" should be a sentence, a sentence is more fitting into the description - maybe i'm wrong?

Reply:
A sentence is fine for TITLE, I think about half of our packages do this. The naming is a bit off, it would be better if the field TITLE was called SUMMARY...

I generally avoid to set the description field, nobody reads that anyways.

Reply:
thanks for the explanation!